tool-box/Dockerfile

```diff
@@ -4,27 +4,27 @@ FROM registry.access.redhat.com/ubi9/ubi:9.3-1610@sha256:66233eebd72bb5baa25190d
 SHELL ["/bin/bash", "-c"]
 
 # renovate: datasource=repology depName=homebrew/openshift-cli
-ARG OC_VERSION=4.15.11
+ARG OC_VERSION=stable-4.15
 # renovate: datasource=github-releases depName=openshift/rosa
-ARG ROSA_VERSION=v1.2.37
+ARG ROSA_VERSION=latest
```

Comment:
would rather keep these pinned to versions, that's the whole point of letting renovate update

Reply:
sure - but the bigger problem is that there's very infrequent releases of this repo